app/templates/application.hbs

```diff
@@ -5,7 +5,10 @@
   {{outlet}}
 </main>
 <aside>
+   
   <p id="detail-list-title">Potential Failures ({{@model.length}})
+    <p>{{input type="text" class="search" placeholder="Search" value=search action="queryPost"}}</p>
```

Comment:
All form elements require an associated `<label>` element. Do you know if this addon accepts the `aria-label` attribute? If we add `aria-label="search potential failures"` to this input, that would suffice. 👍